Sources/TuistGraphTesting/Models/ProjectOptions+TestData.swift

```diff
@@ -25,6 +29,22 @@ extension Project.Options {
             textSettings: textSettings
         )
     }
+
+    public static func testOptions(
```

Comment:
I'm not sure if we want to deprecate old initializers, I had to add an additional one to keep backward compatibility and allow compilation (default values in the functions caused a compilation issue)

I can definitely add a deprecation warning if we decide to adopt this API